tools/local/collect-node-metrics.sh

```diff
@@ -76,5 +159,42 @@ done
 #   measure_timestamp TIMESTAMP
 # );
 
+docker run --rm jbergknoff/postgresql-client "postgresql://qdrant:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:5432/postgres" -c "INSERT INTO chaos_testing (url, version, commit, num_vectors, num_snapshots, measure_timestamp) VALUES $CHAOS_TESTING_VALUES;"
 
-docker run --rm jbergknoff/postgresql-client "postgresql://qdrant:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:5432/postgres" -c "INSERT INTO chaos_testing (url, version, commit, num_vectors, num_snapshots, measure_timestamp) VALUES $PSQL_VALUES;"
+# Assume table:
+# create table chaos_testing_shards (
+#   id SERIAL PRIMARY key,
+#   url VARCHAR(255),
+#   peer_id INT,
+#   shard_id INT,
+#   points_count INT,
+#   state VARCHAR(255),
+#   measure_timestamp TIMESTAMP
+# );
+
+if [ -n "$CHAOS_TESTING_SHARD_VALUES" ]; then
+    docker run --rm jbergknoff/postgresql-client "postgresql://qdrant:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:5432/postgres" -c "INSERT INTO chaos_testing_shards (url, peer_id, shard_id, points_count, state, measure_timestamp) VALUES $CHAOS_TESTING_SHARD_VALUES;"
```

Comment:
I feel like we should eventually switch to something like Python, because this does not seem to be the right way of doing things. It is becoming quite hard to maintain (?) and review this way.

Reply:
I agree. I learn something new about bash almost every time because it's quite different. Things don't work, and they don't give you a clear message about went wrong. 

Exact opposite of Rust.